src/app/content/components/Sidebar/styled/index.tsx

```diff
@@ -11,26 +11,35 @@ import ContentLinkComponent from '../../ContentLink';
 
 export * from './wrapper';
 
-const numberCharacterWidth = 1;
-const numberPeriodWidth = 0.2;
+/* to regenerate these numbers, run this in a rex browser window
+(
+  (element) => 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890.'.split('').reduce(
+    (result, char) => (result[char] = ((element.innerText = char) && element.getBoundingClientRect().width)) && result,
+    {}
+  )
+)(element = document.querySelector('[data-testid=toc] [aria-label="Current Page"] a')
+  .appendChild(document.createElement('span'))
+);
+ */
+const numberCharacterWidth = .7796875;
+const letterCharacterWidth = 1.0375;
```

Comment:
Each number _does_ have the same width in the current font. Interesting.

Reply:
yea letters not so much though. since letters are less common in the sidebar i just went with one of the wider ones